sdk/python/feast/cli.py

```diff
@@ -41,8 +41,10 @@
     registry_dump,
     teardown,
 )
+from feast.utils import maybe_local_tz
 
 _logger = logging.getLogger(__name__)
+warnings.filterwarnings("ignore", category=DeprecationWarning, module="(?!feast)")
```

Comment:
it's just annoying that calling `feast apply` generates million of deprecation warnings from google packages.

Reply:
Should we just upgrade some of these packages?